libs/core/src/lib/table/table.component.scss

```diff
@@ -14,11 +14,6 @@
 	}
 }
 
-// Remove after merging and migrate https://github.com/SAP/fundamental-styles/pull/2192
```

Comment:
Table changes will come with `0.17.0`, so let's uncommit this file as well

Reply:
we brought the latest https://github.com/SAP/fundamental-ngx/blob/7940614e37ba0f49eb746ab2fcb887c98fe83db0/package.json#L116

Reply:
i reverted it